### ISSUE TITLE
Cannot use in a node.js application?

### ISSUE THREAD

**Issue by amandapouget** [davidkudera/google-maps-loader#40]
Installed into a node.js application, opened a file, added require('google-maps'), got this error:
'Google-maps package can be used only in browser'

Could you add some documentation around that? Is it strictly not possible to use this package in a node.js application or are we holding it wrong?

**Comment by davidkudera** [comment]
Hi, it is not possible to do something like that with this library. It's only assynchronous loader for google's own map library.

Maybe [this](https://github.com/moshen/node-googlemaps) library would be better for you. It reiplements google's API calls so you can use it in any environment. 

But good idea with adding it to docs :wink:

**Comment by batjko** [comment]
I'm trying to spyOn this package, in order to unit test a service that uses this library.
But the error mentioned by @mandysimon88 comes up even upon import already, throwing before I can even try to mock it up.

How does one write unit tests for anything using this library?

**Comment by gDelgado14** [comment]
@batjko pardon my ignorance as I don't have much testing experience, but wouldn't something like [PhantomJS](http://phantomjs.org/ ) allow you to test?

**Comment by batjko** [comment]
@gDelgado14 We use jsdom as a mock-browser to render React components into, and Enzyme to do that rendering.
We do that in all our apps and it works fine with everything, usually. So we wouldn't want to have to switch our testing stack, just to cover this one library.

**Comment by davidkudera** [comment]
We could probably move the error message into `load()` function if it helps.

I [plan](https://github.com/Carrooi/Js-GoogleMapsLoader/issues/52) to rewrite this package completely, but I don't have any time right now since I'm leaving my current job. So before that don't you want to create pull request?

**Comment by davidkudera** [comment]
Hello, I added the info about browser only environment to the readme and this package will now immediately tell you if you try to use it in an environment without global `window` object.

Check out the official library from google for usage in nodejs: https://www.npmjs.com/package/@google/maps

Also, I just released a new version of this package 4.1.1. It's now written in typescript and supports await/async.
